File: Unity_project/Assets/Scripts/Constants/PipeEnums.cs
using UnityEngine;

// [action code] | [sender] | [item] | [quantity] | [price] | [message]
public enum EnumType
{
    ActionCode,
    Sender,
    Item
}

public enum ActionCode
{
    TESTMESSAGE,
    TXTMESSAGE,
    SELL,
    CONFIRMSELL,
    ENDCONVARSATION
}

public enum Sender
{
    TEST,
    SMITH,
    BAKER,
    HERBALIST,
    PLAYER,
}
public enum Item
{
    TEST,
    SWORD,
    BREAD,
    WEED
}

File: Pipe/Python/pipe_enums.py
from enum import Enum, auto

class ActionCode(Enum):
    TESTMESSAGE = auto()
    TXTMESSAGE = auto()
    SELL = auto()
    CONFIRMSELL = auto()
    ENDCONVARSATION = auto()

class Sender(Enum):
    TEST = auto()
    SMITH = auto()
    BAKER = auto()
    HERBALIST = auto()
    PLAYER = auto()

class Item(Enum):
    TEST = auto()
    SWORD = auto()
    BREAD = auto()
    WEED = auto()

class EnumType(Enum):
    ActionCode = ActionCode
    Sender = Sender
    Item = Item